monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -136,7 +142,7 @@ def fill_template_config(self, data_stats_filename, **kwargs):
         Args:
             data_stats_filename: filename of the data stats report (generated by DataAnalyzer)
         """
-        pass
+        raise NotImplementedError
```

Comment:
Please add more error message, refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/engines/workflow.py#L293

Thanks.

Reply:
Fixed. Thanks!